Tools-Override/tests.targets

```diff
@@ -52,6 +52,8 @@
     <XunitOptions>$(XunitOptions) -xml $(XunitResultsFileName)</XunitOptions>
 
     <XunitOptions Condition="'$(Performance)'!='true' and '$(Outerloop)' != 'true'">$(XunitOptions) -notrait Benchmark=true</XunitOptions>
+    <!-- Don't run performance tests in parallel, even in "functional" outerloop runs. -->
+    <XunitOptions Condition="'$(Performance)'!='true' and '$(Outerloop)' == 'true' and '$(IncludePerformanceTests)' == 'true'">$(XunitOptions) -parallel none</XunitOptions>
```

Comment:
Just wondering why Performance != true?

Is this because we started to include performance test in regular runs without having to run them using the PerfRunner?